internal/nomad/nomad_test.go

```diff
@@ -666,6 +666,7 @@ func (s *ExecuteCommandTestSuite) SetupTest() {
 }
 
 const withTTY = true
+const unprivileged = false
```

Comment:
Isn't this the opposite meaning of the `privilegedExecution` flag? The value of `unprivileged` is passed to the `ExecuteCommand` function and by reading, I would assume it behaves differently. 

Maybe I just mixing things? Otherwise, I would propose to rename this variable to `privileged` or something similar.

Reply:
I also was unsure about this naming :sweat_smile: 
My argumentation was that for lines like this:
https://github.com/openHPI/poseidon/blob/537030e98d88b19e419aa657e438e8ed79add822/internal/nomad/nomad_test.go#L703-L704
it's easier to read without (having to know /) knowing the value of the constant (as the name is explaining it).

But for line 669 it's the opposite and one may think it's privileged.. I have changed the name to `privileged` :ok_hand:

Reply:
Ah, I see; a probably we might also have for the `withTTY`, have we? I am not aware of a super-nice naming pattern; the only alternative I am aware of are constants (even when just representing a boolean value). Those could indicate to the reader (without knowing the value) what they are meant for. In this case, it would require us to use two constants: One for the privileged execution and one for the unprivileged execution... 
When having a "normal" variable, I liked having the same semantic meaning of the variable name (regardless of content) with the parameter name (just to ease debugging).

Reply:
Like this?

Reply:
Yes, like this :) What do you think about it? Does it combine the better readability in the tests without clashing the argument and parameter names?

Reply:
Yes, I like it too :)

Reply:
Nice, thanks for taking care. Let's merge it and bring it live(?) 🎉